otterdog/eclipse-edc.jsonnet

```diff
@@ -87,15 +87,6 @@ orgs.newOrg('eclipse-edc') {
       workflows+: {
         default_workflow_permissions: "write",
       },
-      environments: [
```

Comment:
is this intentional?

at least legacy gh pages are actived for that repository and an environment github-pages will be created automatically.

Reply:
@netomi yes, I removed this part to uniform the configurations, the azure environment is not used anymore (for 2 years) and the github-pages, as you said is created automatically

Reply:
the github-pages environment is created automatically, but its good to have it in the config to avoid warnings and still configure the environment to your needs. If it is missing, otterdog will just skip it.

Reply:
so do we need to add the environment configuration to all the other repos that have gh pages enabled?

Reply:
to get rid of the warnings during validation, yes.

Reply:
@netomi done

Reply:
perfect ty.